core/src/main/mima-filters/0.20.backwards.excludes

```diff
@@ -1,10 +1,90 @@
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$AssignWithOffset$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$AssignOffsetsForTimes")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$AssignOffsetsForTimes$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Commit")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Commit$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Committed")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Committed$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$ConsumerMetrics")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$ConsumerMetrics$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Messages")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Messages$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$NoPollResult")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Poll")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Poll$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Revoked")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Revoked$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$RequestMessages")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$RequestMessages$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$RequestMetrics$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Seek")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Seek$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Stop$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Subscribe")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$Subscribe$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$SubscribePattern")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$SubscribePattern$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$Internal$SubscriptionRequest")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$ListenerCallbacks")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$ListenerCallbacks$")
+ProblemFilters.exclude[MissingClassProblem]("akka.kafka.KafkaConsumerActor$WrappedAutoPausedListener")
```

Comment:
Such wildcard would be too wide as it would ignore the `props` method and the `Stop` object that has been not moved.

But you can use the following shrink this list substantially:
```
ProblemFilters.exclude[MissingClassProblem("akka.kafka.KafkaConsumerActor$Internal$*")